loopchain/channel/channel_service.py

```diff
@@ -417,14 +417,19 @@ def _handle_exception(future: asyncio.Future):
                 else:
                     logging.warning(f"Waiting for next subscribe request...")
 
-        subscribe_event = asyncio.Event()
         utils.logger.spam(f"try subscribe_call_by_citizen target({ChannelProperty().rest_target})")
-
+        subscribe_event = asyncio.Event()
+        subscribe_params = {
+            "block_height": self.__block_manager.blockchain.block_height,
+            "event": subscribe_event
+        }
         # try websocket connection, and handle exception in callback
-        asyncio.ensure_future(self.__node_subscriber.start(
-            block_height=self.__block_manager.blockchain.block_height,
-            event=subscribe_event,
-        ), loop=MessageQueueService.loop).add_done_callback(_handle_exception)
+        task = asyncio.ensure_future(
+            self.__node_subscriber.start(**subscribe_params),
```

Comment:
How about pass `block_height` and `event` as a keyword args, not the unpacked dict?

Reply:
Seems reasonable. Updated!